deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,205 @@
+locals {
+  app_image = "${module.ecr_repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group" "ecs_service" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgEcsService"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+#
+# ALB Resources
+#
+resource "aws_lb" "app" {
+  name            = "alb${var.environment}"
```

Comment:
Similar comment with regard to including the name of the service behind the load balancers (e.g., `alb${var.environment}App`).